packages/use-query-params/README.md

```diff
@@ -0,0 +1,34 @@
+# `@scaleway/use-query-params`
+
+## A tiny hooks to handle use-query-params
+
+## Install
+
+```bash
+$ yarn add @scaleway/use-query-params
+```
+
+## Usage
+
+```js
+import React from 'react'
```

Comment:
Might be good to add an example with a custom updater

Reply:
Done ;)